ecl/agentexec/agentexec.cpp

```diff
@@ -248,9 +248,9 @@ class WaitThread : public CInterfaceOf<IPooledThread>
                     params.push_back({ "graphName", graphName.get() });
                     jobName.append('-').append(graphName);
                 }
-                bool deleteJobs = queryComponentConfig().getPropBool("@deleteJobs", true);
+                KeepK8sJobs keepK8sJobs = translateKeepJobs(queryComponentConfig().queryProp("@keepJobs"));
```

Comment:
I'm wondering whether - for settings that are common to multiple components - code like this should not be moved into workunit.cpp, rather than having all callers of the workunit functions having to make exactly the same calls to decide what parameters to pass?

Reply:
I was wondering whether to change to pass in component config - and have the common funcs query the options of interest.
There may be others to come..

Reply:
Do you even need to pass in the component config - you can call queryComponentConfig() from within workunit.cpp

Reply:
yes, I've changed to be like that.